webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const EmptyState = (props) => {
+  const {
+    icon = 'add-circle-o',
+    header,
+    description,
+    documentation: {
+      label = __('For more information please see'),
+      buttonLabel = __('Documentation'),
+      url,
+    },
+    action,
+    secondayActions,
+  } = props;
+
+  return (
+    <PfEmptyState>
+      <PfEmptyState.Icon type="pf" name={icon} />
+      <PfEmptyState.Title>{header}</PfEmptyState.Title>
+      <PfEmptyState.Info>{description}</PfEmptyState.Info>
+      {url &&
+        <PfEmptyState.Help>
+         {label}
+          <a href={url}> {buttonLabel} </a>
```

Comment:
Nitpick:
this indend doesn't put any space between the text and the link. Spaces in `<a>` work but they look weird on hover:
![screenshot from 2018-07-09 15-57-09](https://user-images.githubusercontent.com/1186631/42454858-ef55cf10-8390-11e8-9f15-769dfae3dfa1.png)

Please indent correctly and remove spaces form the anchor. `{" "}` should help with preserving the space in between.